recipe/meta.yaml

```diff
@@ -24,55 +24,62 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - packaging
   run:
    - python
-   - applaunchservices >=0.3.0  # [osx]
+   - python.app                        # [osx]
+   # This is here to work around a bug in mamba
+   - ptyprocess >=0.5                  # [win]
+   - aiohttp >=3.9.3
+   - applaunchservices >=0.3.0         # [osx]
+   - asyncssh >=2.14.0,<3.0.0
    - atomicwrites >=1.2.0
    - chardet >=2.0.0
    - cloudpickle >=0.5.0
    - cookiecutter >=1.6.0
    - diff-match-patch >=20181111
+   - importlib-metadata >=4.6.0        # [py<3.10]
    - intervaltree >=3.0.2
-   - ipython >=8.12.2,<8.13.0  # [py==38]
+   - ipython >=8.12.2,<8.13.0          # [py==38]
    - ipython >=8.13.0,<9.0.0,!=8.17.1  # [py>38]
    - jedi >=0.17.2,<0.20.0
-   - jellyfish >=0.7          # [not (osx and arm64 and py<312)]
+   - jellyfish >=0.7                   # [not (osx and arm64 and py<312)]
    # Tests fail with "jellyfish 1.0.1 is not supported on this platform"
-   - jellyfish >=0.7,!=1.0.1  # [osx and arm64 and py<312]
+   - jellyfish >=0.7,!=1.0.1           # [osx and arm64 and py<312]
    - jsonschema >=3.2.0
    - keyring >=17.0.0
    - nbconvert >=4.0
    - numpydoc >=0.6.0
-   - paramiko >=2.4.0  # [win]
    - parso >=0.7.0,<0.9.0
    - pexpect >=4.4.0
    - pickleshare >=0.4
-   # This is here to work around a bug in mamba
-   - ptyprocess >=0.5  # [win]
    - psutil >=5.3
+   - pygithub >=2.3.0
    - pygments >=2.0
-   - pylint >=2.5.0,<3.1
+   - pylint >=3.1,<4
    - pylint-venv >=3.0.2
-   - python-lsp-black >=2.0.0,<3.0.0
    - pyls-spyder >=0.4.0
-   - pyqt >=5.10,<5.16
-   - pyqtwebengine >=5.10,<5.16
-   - python.app  # [osx]
-   - python-lsp-server >=1.10.0,<1.11.0
-   - pyxdg >=0.26  # [linux]
-   - pyzmq >=22.1.0
+   - pyqt >=5.15,<5.16
+   - pyqtwebengine >=5.15,<5.16
+   - python-lsp-black >=2.0.0,<3.0.0
+   - python-lsp-server >=1.12.0,<1.13.0
+   - pyuca >=1.2.0
+   - pyxdg >=0.26                      # [linux]
+   - pyzmq >=24.0.0
    - qdarkstyle >=3.2.0,<3.3.0
    - qstylizer >=0.2.2
-   - qtawesome >=1.2.1
-   - qtconsole >=5.5.1,<5.6.0
-   - qtpy >=2.1.0
+   - qtawesome >=1.3.1,<1.4.0
+   - qtconsole >=5.6.0,<5.7.0
+   - qtpy >=2.4.0
    - rtree >=0.9.7
    - setuptools >=49.6.0
    - sphinx >=0.6.6
-   - spyder-kernels >=2.5.0,<2.6.0
+   - spyder-kernels >=3.0.0,<3.1.0
+   - superqt >=0.6.2,<1.0.0
    - textdistance >=4.2.0
    - three-merge >=0.1.1
    - watchdog >=0.10.3
+   - yarl >=1.9.4
   run_constrained:
     - menuinst >=1.4.17 
```

Comment:
This is outdated, and not sure whether it should be here anyway. seems to be required to make an installer. (our installer? not sure.) https://github.com/spyder-ide/spyder/blob/64ff77ca98a7b8bb3f7eb759ab3c909750c4f0e5/installers-conda/build_installers.py#L144

Reply:
I mean menuinst

Reply:
Oh, gotcha. I wondered about that one as well. Since you pointed it out, I updated the version to match upstream. Do we think it should still be included? I imagine it's intended to be used to install the desktop icons into OS-specific applications menus.

Reply:
cool, sounds like it's appropriate here